pypeit/telescopes.py

```diff
@@ -1,171 +1,193 @@
+                                               latitude=loc.lat.to(units.deg).value,
+                                               elevation=loc.height.to(units.m).value,
                                                fratio=15,
                                                diameter=10,
                                                eff_aperture=72.3674)
 
 
 class MagellanTelescopePar(TelescopePar):
     def __init__(self):
+        loc = EarthLocation.of_site('Las Campanas Observatory')
         super(MagellanTelescopePar, self).__init__(name='MAGELLAN',
-                                               longitude=70.6858,
-                                               latitude=-29.0283,
-                                               elevation=2516.0,
+                                               longitude=loc.lon.to(units.deg).value,
+                                               latitude=loc.lat.to(units.deg).value,
+                                               elevation=loc.height.to(units.m).value,
                                                diameter=6.5)
 
 class ShaneTelescopePar(TelescopePar):
     def __init__(self):
+        loc = EarthLocation.of_site('Lick Observatory')
         super(ShaneTelescopePar, self).__init__(name='SHANE',
-                                                longitude=121.6428,
-                                                latitude=37.3413889,
-                                                elevation=1283.0,
+                                                longitude=loc.lon.to(units.deg).value,
+                                                latitude=loc.lat.to(units.deg).value,
+                                                elevation=loc.height.to(units.m).value,
                                                 diameter=3.05,
                                                 eff_aperture=6.3617)
 
                                 
 class WHTTelescopePar(TelescopePar):
     def __init__(self):
+        loc = EarthLocation.of_site('Roque de los Muchachos')
         super(WHTTelescopePar, self).__init__(name='WHT',
-                                              longitude=17.8947,
-                                              latitude=26.7636,
-                                              elevation=2396.0,
+                                              longitude=loc.lon.to(units.deg).value,
+                                              latitude=loc.lat.to(units.deg).value,
+                                              elevation=loc.height.to(units.m).value,
                                               diameter=4.2)
                                 
 class APFTelescopePar(TelescopePar):
     def __init__(self):
         super(APFTelescopePar, self).__init__(name='APF',
-                                              longitude=121.642778,
+                                              longitude=-121.642778,
```

Comment:
Can we use `loc = EarthLocation.of_site('Lick Observatory')`?

Reply:
Given that it's predominantly used by planet hunters (I presume), I didn't dare change it from the default value... I was being overly cautious, perhaps... Happy to update this...